test/90_1_prm/test_appendix_g_prm.rb

```diff
@@ -1223,6 +1233,49 @@ def check_if_vav_chiller(model, sub_text)
     end
   end
 
+  # Check if model uses standard VAV boxes of FP boxes
+  # @param model [OpenStudio::Model::Model] OpenStudio model
+  # @param energy_source [String] Energy source used for heating
+  def check_terminal_type(model, energy_source, mod_str)
+    model.getAirLoopHVACs.each do |airloop|
+      airloop.thermalZones.each do |zone|
+        zone.equipment.each do |equip|
+          expected_results = false
+          if equip.to_AirTerminalSingleDuctVAVReheat.is_initialized
+            expected_results = true if energy_source != 'Electric'
+            assert(expected_results, "Standard VAV boxes are not expected for #{mod_str}.")
+          elsif equip.to_AirTerminalSingleDuctParallelPIUReheat.is_initialized
+            expected_results = true if energy_source == 'Electric'
+            terminal = equip.to_AirTerminalSingleDuctParallelPIUReheat.get
+            assert(expected_results, "Fan powered boxes are not expected for #{mod_str}.")
+            # check primary flow fraction
+            min_pri_flow_frac = terminal.minimumPrimaryAirFlowFraction.get
+            assert(min_pri_flow_frac == 0.3, "The FPB minimum primary air flow fraction is #{min_pri_flow_frac} instead of 0.3 for #{mod_str}.")
+            # check secondary flow fraction
+            check_secondary_flow_fraction(terminal, mod_str)
+          end
+        end
+      end
+    end
+  end
```

Comment:
New method that checks if the correct type of VAV terminal is used based on the baseline energy type.